RegisterAPI/DataAccess/CassandraHandler.cs

```diff
@@ -0,0 +1,72 @@
+﻿/// <summary>

```

Comment:
Use this Library, it has a framework for Cassandra layer : https://github.com/atulsagarEngaze/Core